package.json

```diff
@@ -7,7 +7,8 @@
     "test": "npx codeceptjs run --steps",
     "test:api": "MOCHAWESOME_REPORTFILENAME=api npx codeceptjs run --grep @api-tests --reporter mocha-multi --verbose",
     "test:smoke": "MOCHAWESOME_REPORTFILENAME=smoke npx codeceptjs run --grep @smoke-tests --reporter mocha-multi --verbose",
-    "test:functional": "MOCHAWESOME_REPORTFILENAME=functional npx codeceptjs run --invert --grep @smoke-tests --reporter mocha-multi"
+    "test:functional": "MOCHAWESOME_REPORTFILENAME=functional npx codeceptjs run --invert --grep @smoke-tests --reporter mocha-multi",
+    "test:e2e": "MOCHAWESOME_REPORTFILENAME=functional npx codeceptjs run --grep @claim-tests --reporter mocha-multi"
```

Comment:
do we need that new command?

Reply:
I added it for easy of use for running just the e2e happy path test locally - mainly for my benefit